includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -8973,7 +8978,14 @@ class AMP_Allowed_Tags_Generated {
 		),
 		'html' => array(
 			array(
-				'attr_spec_list' => array(),
+				'attr_spec_list' => array(
+					'data-amp-autocomplete-opt-in' => array(
+						'blacklisted_value_regex' => 'false',
+						'value' => array(
+							'false',
```

Comment:
Should this be `false` or `'false'`? I noticed that some of these boolean values have been wrapped in quotes for quite some time now; just wondering if there's any reason behind it.

Reply:
No because it's the HTML attribute value which only involves strings.

Reply:
Ah OK thanks for clarifying.

Reply:
In any case, this attribute spec is somewhat of a hack in that it requires a value that can never be allowed. It's also only applicable for AMP4email, so it should never intersect with us here.